File: frontend/src/features/actividades/generar_memo_escritura/models/generar_memo_escritura.ts
export interface GenerarMemoEscritura {
  id_generar_memo_escritura: number;
  id_expediente: number;
  enviar_reparo: boolean;
  observaciones: string | null;
  is_active: boolean;
  row_status: boolean;
  created_by: number;
  created_date: string;
  modified_by: number | null;
  modified_date: string | null;
}

File: backend/Multibanca.Domain.Models/Multibanca/generar_memo_escritura.cs
using Common.Domain.Models;

namespace Multibanca.Domain.Models.Multibanca
{
    public class generar_memo_escritura : base_auditoria
    {
        public int id_generar_memo_escritura { get; set; }
        public long id_expediente { get; set; }
        public bool enviar_reparo { get; set; }
        public string? observaciones { get; set; }
    }
}
